app/models/course-participation.ts

```diff
@@ -0,0 +1,21 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+import type CourseModel from './course';
+import type CourseStageModel from './course-stage';
+import type UserModel from './user';
+import type LanguageModel from './language';
+
+export default class CourseParticipationModel extends Model {
+  @belongsTo('course', { async: false, inverse: null }) declare course: CourseModel;
+  @belongsTo('course-stage', { async: false, inverse: null }) declare currentStage: CourseStageModel;
+  @belongsTo('user', { async: false, inverse: 'courseParticipations' }) declare user: UserModel;
+  @belongsTo('language', { async: false, inverse: null }) declare language: LanguageModel;
+
+  @attr('date') declare completedAt: Date;
+  @attr('date') declare lastActivityAt: Date;
```

Comment:
Let's remove this, since it isn't being returned by the backend. Same for `started_at`